java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedJobsMultipartManagerIT.java

```diff
@@ -55,7 +55,7 @@
 /* Tests are disabled because functionality is obsolete and overall test runtime
  * is very long and expensive. */
 @Deprecated
-@Test(groups = { "encrypted" }, enabled = false)
+@Test(groups = { "expensive" }, enabled = false)
```

Comment:
Your suggestion was factored into determining which `integration-tests` and `sub-tests` within other integration-tests will be defined as expensive (nightly). Also, I believe the reasoning behind your suggestion was because the class was deprecated and comprised of **heavy** operations.

Reply:
The discussion was made [here](https://github.com/joyent/java-manta/pull/492#issuecomment-475365013).